lib/rails/dom/testing/assertions/selector_assertions.rb

```diff
@@ -178,6 +178,7 @@ def assert_select(*args, &block)
             nest_selection(matches, &block) if block_given? && !matches.empty?
           end
         end
+        deprecate assert_select: "assert_select will be renamed to assert_dom in an upcoming release"
```

Comment:
Do we think that `assert_css_select` or `assert_dom_select` is a more communicative new name than `assert_dom`?

Reply:
Don't think so, no.